docs/mozillavpn.json

```diff
@@ -111,8 +111,8 @@
     },
     "2.12.0": {
       "build": null,
-      "date": "2022-12-15",
-      "platforms": ["MACOS", "WINDOWS"]
+      "date": "2022-12-16",
```

Comment:
do not change the date.

Reply:
@birdsarah  you happen to know what this date is for?

Reply:
it's just for record keeping it has no functional impact. it should reflect the actual day of release but given that there's multiple release dates and we don't have a feature for that we just pick "the best one" which I'd say is usually when Mac and Windows go out.